community/planning/batch_tracking_store.md

```diff
@@ -31,8 +31,8 @@ pub trait BatchTrackingStore {
     fn get_batch_status(
         &self,
         id: &str,
-        service_id: Option<&str>,
-    ) -> Result<BatchStatus, BatchTrackingStoreError>;
+        service_id: &str,
+    ) -> Result<Option<BatchStatus>, BatchTrackingStoreError>;
```

Comment:
Do we think the schema will also be affected by the scope ID work happening?

Reply:
I don't believe so. I think all that happens before it gets here